src/app/store.js

```diff
@@ -1,12 +1,20 @@
+import { useDispatch as useReduxDispatch, useSelector as useReduxSelector } from 'react-redux';
 
 import {
   configureStore
 } from '@reduxjs/toolkit';
-import userReducer from '../features/user/userSlice';
+import userReducer from '../features/slices/user';
+import itemReducer from '../features/slices/item'
 
-export const store = configureStore({
+const store = configureStore({
   reducer: {
     user: userReducer,
-
+    item: itemReducer
   },
 });
+
+const { dispatch } = store;
+const useSelector = useReduxSelector;
+const useDispatch = () => useReduxDispatch();
```

Comment:
What are those hooks (useReduxDispatch and useReduxSelector) ?

Reply:
Yes, it's; since the store file is the config file for the redux, I preferred to import/export all redux hooks/functions from that file “store” instead of importing them every time in every file from ‘react-redux’; it’s a way to build a clean and organized code.
It’s a way of writing code. If you prefer to do it traditionally, I will do them

Reply:
No no its fine really, but I cant even find those hooks in the docs, so I was wondering where they came from